rclpy/rclpy/node.py

```diff
@@ -756,6 +756,22 @@ def _set_parameters_atomically(
             self._parameter_event_publisher.publish(parameter_event)
 
         return result
+    
+    def add_on_set_paramters_callback(self,callback:Callable([List[Parameter],SetParametersResult])):
+        """add the callback to list"""
+
+        prevCallBack = self._parameters_callback
+        prevCallBack.insert(0,callback)
+        self._parameters_callback = prevCallBack
+
+    
+    def remove_on_set_parameters_callback(self,callback:Callable[List[Parameter],SetParametersResult]):
+        """remove callback from list"""
+        try:
+            self._parameters_callback.remove(callback)
+        except ValueError:
+            print("Callback doesn't exist")
```

Comment:
I guess it's okay to let the exception go (ie. remove the `try-except`), instead of catching it and printing an error message.